plugins/together/newhelm/annotators/llama_guard_annotator.py

```diff
@@ -8,11 +8,12 @@
 
 from pydantic import BaseModel
 
+from newhelm.annotation import Annotation
 from newhelm.base_annotator import BaseAnnotator
 from newhelm.single_turn_prompt_response import PromptInteraction
 
 
-class LlamaGuardAnnotation(BaseModel):
+class LlamaGuardAnnotation(Annotation):
```

Comment:
As of #118, annotations don't inherit from Annotation. Annotation is the wrapper around TypedData. Was something complaining about this?

Reply:
Yeah, I just merged in those changes and it complained. But it's all fixed now.